### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by vxav**
This PR:

- Updates the README

### Checklist

- [x] Update changelog in CHANGELOG.md.
- [x] Make sure `values.yaml` and `values.schema.json` are valid.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,9 +2,30 @@
...
-* Manual chart creation (not kustomize generated).
-* Basic crd creation check in values.yaml.
+The image is currently built manually from a specific commit since the only release is outdated and `main` is the development branch. The capvcd helm chart is based on the output of `kustomize build` in the following folder:
+
+* Repo: https://github.com/vmware/cluster-api-provider-cloud-director/tree/main/config/default
+* Commit: 003336f85f44312252553dc1369faf409e18c10b
+
+## How to use
+
+There are no credentials in the CAPVCD controller. The credentials are stored in a secret in a namespace that is referenced in the cluster chart.
```
Comment (kopiczko) [inline README.md:14]: This is a bit confusing to me. You mean that credentials are supposed to be provided as an external secret right?

File: README.md [hunk 2]
```diff
@@ -2,9 +2,30 @@
...
+
+* Install cluster API v1.1.3.
+
+`clusterctl init --core cluster-api:v1.1.3 -b kubeadm:v1.1.3 -c kubeadm:v1.1.3`
```
Comment (kopiczko) [inline README.md:18]: Do you plan to keep this version up to date?

**Review by kopiczko — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (vxav) [reply README.md:18]: I'm putting it in the README for now as I'm the only tinkering with it but we'll make it more generic later on. The goal is to be consistent across providers (on-prem at least) down the line.

File: README.md (hunk 1 above)
Comment (vxav) [reply README.md:14]: The CAPVCD controller has no creds at all, it just runs the logic. Creds are specified for each cluster https://github.com/giantswarm/cluster-cloud-director/blob/main/helm/cluster-cloud-director/templates/vcdcluster.yaml#L25.

**Pull request merged**